.github/workflows/requirement-check.yml

```diff
@@ -59,3 +59,21 @@ jobs:
           fi
         done < $PACKAGE_TO_UPDATE
 
+  check_resolution:
+    name: Check Package Installation
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream9
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
I think 0 means the whole history, but here you only need a checkout which I think is the default:
```suggestion
```

Reply:
Yes, I'm going to remove the checkout, we just need to check the current HEAD